src/UI/Components/ServiceInstanceForm/Helpers/createFormState.ts

```diff
@@ -93,9 +101,22 @@ export const createEditFormState = (
         case "Textarea":
         case "TextList":
         case "Text": {
-          acc[curr.name] = curr.type.includes("dict")
-            ? stringifyDict(originalAttributes?.[curr.name])
-            : cloneDeep(originalAttributes?.[curr.name]);
+          if (curr.type === "int?" || curr.type === "float?") {
```

Comment:
I'm confused, if we have a Text field, or TextList, how is it we have the possibility that we are dealing with integers or floats? is it purely to cover the empty state `""`?